ansible/roles/riakcs/configure/tasks/main.yml

```diff
@@ -56,6 +57,14 @@
   notify: "restart {{ name }}"
   when: drop_conf != ""
 
+- name: "Create {{ name }} log dir"
+  file: >
+    path="{{ log_path | dirname }}"
+    state=directory
+    owner={{ name }}
+    group=riak
+  when: log_path != ''
```

Comment:
rather than adding another task here you could add an entry to `data_dirs` to the configure role for each of [riak](https://github.com/dimagi/commcarehq-ansible/blob/dm/riak-config/ansible/deploy_riakcs.yml#L82-L86), [riak-cs](https://github.com/dimagi/commcarehq-ansible/blob/dm/riak-config/ansible/deploy_riakcs.yml#L110), and [stanchion](https://github.com/dimagi/commcarehq-ansible/blob/dm/riak-config/ansible/deploy_riakcs.yml#L104)